crates/pallets/starknet/src/lib.rs

```diff
@@ -736,6 +736,23 @@ pub mod pallet {
 
             let transaction = Self::convert_runtime_calls_to_starknet_transaction(call.clone())
                 .map_err(|_| InvalidTransaction::Call)?;
+
```

Comment:
Add a lil comment explaining that transactions v0 don't have Nonce or validation rules.